File: Programmers/Inho/Level01/수박수박수박수박수박수.cs
public class Solution
{
    public string solution(int n)
    {
        string answer = "";
        string s1 = "수";
        string s2 = "박";
        for (int i = 0; i < n; i++)
        {
            answer += (i % 2 == 0) ? s1 : s2;
        }
        return answer;
    }
}

File: Programmers/Inho/C++/Level01/수박수박수박수박수박수.cpp
#include <string>
#include <vector>

using namespace std;

string solution(int n) {
    string answer = "";
    string su = "수";
    string park = "박";
    for(int i = 0;i < n;i++){
        if(i%2==0)
            answer+=su;
        else
            answer+=park;
    }
    return answer;
}